task/02-numbers-tasks.js

```diff
@@ -202,7 +229,14 @@ function roundToPowerOfTen(num, pow) {
  *   17 => true
  */
 function isPrime(n) {
-    throw new Error('Not implemented');
+    if (n % 2 !== 0 && n > 2) {
+        for (let i = n - 2; i > 2; i -= 2) {
+            if (n % i === 0) {
+                return false;
+            }
+        }
+    }
+    return (n % 2 !== 0 || n === 2);
```

Comment:
It's better to put the condition for n === 2 and for the rest even numbers in the very beginning. The earlier you can return the answer clearly, the better